react/ProductLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link } from 'vtex.render-runtime'
+import { useCssHandles } from 'vtex.css-handles'
+import { ModalContext } from 'vtex.modal-layout'
+import { useProduct } from 'vtex.product-context'
+import React, { useState, useEffect } from 'react'
+
+import { AvailableContext } from './modules/mappings'
+import interpolateLink from './modules/interpolateLink'
+
+const { useModalDispatch } = ModalContext
+
+interface Props {
+  label: string
+  href: string
+}
+
+const CSS_HANDLES = ['container', 'link'] as const
+
+export default function StoreLink(props: Props) {
```

Comment:
Wouldn't it be better if the name of this function were `ProductLink`?

Reply:
Oh, forgot to change this